src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -386,7 +385,7 @@ export default class SignUp extends Component {
               </div>
               <div style={{ width: '304px', margin: '10px auto 10px' }}>
                 <Recaptcha
-                  sitekey={CAPTCHA_KEY}
```

Comment:
What does it throw when `sitekey=CAPTCHA_KEY` ?

Reply:
It only works when we click the signup button on the login page.
It does not render when we reload the page.

Reply:
I meant to remove the `{ }` and test

Reply:
Syntax error: D:/GitHub/accounts.susi.ai/src/components/Auth/SignUp/SignUp.react.js: JSX value should be either an expression or a quoted JSX text (389:27)

Reply:
:thinking: 

Okay then, can you think of some other approach? Exposing the key in the code is not good as it's to be read from the config.

Reply:
@anshumanv i have another approach it follows your basic idea of keeping the key out of the code .